backend/src/router.ts

```diff
@@ -181,6 +195,11 @@ router.post("/openai/chat", async (req, res) => {
             chatResponse.defenceInfo.isBlocked =
               chatResponse.defenceInfo.isBlocked ||
               openAiReply.defenceInfo.isBlocked;
+
+            // combine blocked reason
+            chatResponse.defenceInfo.blockedReason =
+              chatResponse.defenceInfo.blockedReason ||
+              openAiReply.defenceInfo.blockedReason;
```

Comment:
This is interesting, does this combine both strings?

Reply:
should be if the user input was blocked, it will show that message, otherwise if the bot response was blocked it will show that message.

Reply:
ah right! Yeah that works for me 👍